test/SnowballStemmer_simple.dart

```diff
@@ -0,0 +1,15 @@
+import '../lib/SnowballStemmer.dart';
```

Comment:
Not sure we'd need this file. Might be nice for experimenting whilst building, but don't think this should be part of the package.

Reply:
Moved this to bin/ per Dart conventions.  Not sure it's worth having, but I think it might be since it's small.

Reply:
(Or perhaps it could be expanded with a switch to use either stemmer so that there's one binary wrapper for the whole package)